app/assets/javascripts/admin/controllers/admin-user-index.js.es6

```diff
@@ -55,21 +57,65 @@ export default Ember.Controller.extend(CanCheckEmails, {
     anonymize() { return this.get('model').anonymize(); },
     destroy() { return this.get('model').destroy(); },
 
+    toggleUsernameEdit() {
+      this.set('userUsernameValue', this.get('model.username'));
+      this.toggleProperty('editingUsername');
+    },
+
+    saveUsername() {
+      const self = this;
+      const old_username = self.get('model.username');
+
+      self.set('model.username', self.get('userUsernameValue'));
+      return ajax(`/users/${old_username.toLowerCase()}/preferences/username`, {
+        data: { new_username: this.get('userUsernameValue') },
+        type: 'PUT'
+      }).catch(function(e) {
```

Comment:
We prefer to use [Arrow functions](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions) in our code base.